bin/oneffs.ml

```diff
@@ -49,6 +51,41 @@ let get_info block_size file =
   | None ->
     print_endline "Unset."
 
+let crc_of_file filename =
+  let ic = open_in filename in
+  let finally () = close_in ic in
+  Fun.protect ~finally @@ fun () ->
+  let buf = Bytes.create 0x7ff in
+  let rec go crc = match input ic buf 0 (Bytes.length buf) with
+    | 0 -> crc
+    | len -> go (Checkseum.Crc32.digest_bytes buf 0 len crc)
+    | exception End_of_file -> crc in
+  go Checkseum.Crc32.default
```

Comment:
A bit a shame to read the file twice :/

Reply:
Depending on what you want but I prefer to stream the whole computation than load the entire file to compute the crc and keep it to blit it into the OneFFS image.